source/getting-started/concepts-terminology.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "Concepts and terminology"
+description: "User explanation of Home Assistant basics"
+---
+Now you're in Home Assistant, let's go over important parts of Home Assistant and what they do.
+
+### Dashboards
```

Comment:
One level to much

```suggestion
## Dashboards

```

PS: It is a good practice to add a new line after a header (common Markdown styling).